fred2/sexp_tree.h

```diff
@@ -27,13 +27,15 @@
 #define SEXPT_UNKNOWN	0x0002
 
 #define SEXPT_VALID		0x1000
-#define SEXPT_TYPE_MASK	0x00ff
+#define SEXPT_TYPE_MASK	0x0fff
```

Comment:
It's needed because `SEXPT_CONTAINER` and `SEXPT_MODIFIER` occupy an additional "column" in the hexadecimal number.

This is going to make things more confusing because the `sexp_node` struct has the type divided up between the first two bytes and the last two bytes, whereas `tree_node` now has the type divided up between the first one byte and the last three bytes.  This is such a fundamental change that I would recommend pulling it out into its own separate PR, in case of unexpected side effects.

Reply:
Ok, good to know. These definitions will go into what'll become the first containers PR of a series.

I'm confused, though -- aren't `0x00ff` and `0x0fff` just two bytes?

Reply:
Er yeah, they're just two bytes.  So change "byte" to "nibble" in what I wrote above.